Mathlib/AlgebraicTopology/SimplicialSet/Basic.lean

```diff
@@ -106,6 +106,12 @@ def _root_.SSet.yonedaEquiv (X : SSet.{u}) (n : SimplexCategory) :
     (standardSimplex.obj n ⟶ X) ≃ X.obj (op n) :=
   yonedaCompUliftFunctorEquiv X n
 
+/-- The unique non-degenerate `n`-simplex in `Δ[n]`. -/
+def idSimplex (n : ℕ) : Δ[n] _[n] := yonedaEquiv Δ[n] [n] (𝟙 Δ[n])
```

Comment:
As this is in the `standardSimplex` namespace, `id` could be a good enough name:
```suggestion
def id (n : ℕ) : Δ[n] _[n] := yonedaEquiv Δ[n] [n] (𝟙 Δ[n])
```